test/index.test.js

```diff
@@ -167,30 +134,30 @@ describe('index', () => {
         };
 
         beforeEach(() => {
-            breakRunMock.runCommand.yieldsAsync(null, fakeStopResponse, fakeStopResponse.body);
+            requestMock.yieldsAsync(null, fakeStopResponse, fakeStopResponse.body);
         });
 
         it('calls breaker with correct config', () => (
             executor.stop({
                 buildId: testBuildId
             }).then(() => {
-                assert.calledOnce(breakRunMock.runCommand);
-                assert.calledWith(breakRunMock.runCommand, deleteConfig);
+                assert.calledWith(requestMock, deleteConfig);
+                assert.calledOnce(requestMock);
             })
         ));
 
         it('returns error when breaker does', () => {
             const error = new Error('error');
 
-            breakRunMock.runCommand.yieldsAsync(error);
+            requestMock.yieldsAsync(error);
```

Comment:
I don't think we should mock `circuit-fuses` any more than we would mock `async`, if we used it. Circuit breakers represent and important run time operation of the module. By mocking it, we lose the ability to make sure that the underlying library calls that the breaker is handed (e.g. `request`, `github.*`, `docker.*`) are called properly.
I don't think we need to have a complete test of breaker functionality, but we should make sure that the code we expect to execute is executed in the expected way.